articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -183,10 +231,19 @@ To create a **remediation task**, follow these steps:
    > An alternate way to open the **remediation task** page is to find and select the policy from
    > the **Compliance** page, then select the **Create Remediation Task** button.
 
-1. On the **New remediation task** page, filter the resources to remediate by using the **Scope**
+1. On the **New remediation task** page, optional remediation settings are shown: 
+
+1. **Failure Threshold percentage** used to specifying whether the remediation task should fail if the percentage of failures exceeds the given threshold. Provided as a number between 0 to 100. By default, the failure threshold is 100%. 
+2. **Resource Count** determines how many non-compliant resources to remediate in a given remediation task. The default value is 500 (the previous limit). The maxium number of is 10,000 resources.
+3. **Parallel Deployments** determines how many resources to remediate at the same time. The allowed values are 1 to 15 resources at a time. The default value is 10. 
+
+   > [!NOTE]
+   > These settings cannot be changed once the remediation task has started.
+ 
+1.  On the same page, filter the resources to remediate by using the **Scope**
    ellipses to pick child resources from where the policy is assigned (including down to the
    individual resource objects). Additionally, use the **Locations** dropdown list to further filter
-   the resources. Only resources listed in the table will be remediated.
+   the resources. The table will only show a subset (top 250) resources. 
```

Comment:
Maybe instead of this sentence, just add a note saying that if the number of non-complaint resources that matches the selected filter is too large, the table will only show the _first_ (I don't like the word top, as it implies some sort of order) 250 non-compliant resources.
Alternatively just drop this part altogether. We're already show an info box in the UX in this case, there's no need to cover it here. It's too much information IMO.

Reply:
Dropped.